### PR TITLE
fix(dedicatedcloud-datacenter-backup-new): cleanup ui flow 

### PR DESCRIPTION
**Pull request by ganeshkumar1989**
- css fix on new backup page
- preview mode enable for offer selection
- spinner addition for order in progress
- conditions url addition
- bug fixes for API calls

ref: MANAGER-4442

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/termsOfUse/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "dedicatedCloud_datacenter_backup_terms_of_use_description1_new": "Êtes-vous sûr(e) de vouloir activer la sauvegarde dans le datacenter {{datacenter}} ?",
   "dedicatedCloud_datacenter_backup_terms_of_use_description2_new": "Attention, vous serez facturé(e) selon le nombre et la taille des machines virtuelles sauvegardées.",
   "dedicatedCloud_datacenter_backup_terms_of_use_tariff_url": "Consulter en détail la nouvelle gamme Veeam Backup",
-  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions": "J’ai lu et j’approuve les conditions générales d’utilisation",
+  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions_text": "J’ai lu et j’approuve les",
+  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions_url": "conditions générales d’utilisation",
```
Comment (cbourgois) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/termsOfUse/translations/Messages_fr_FR.json]: It doesn't work for all languages, in German we could have the verb at the end of the sentence.

Comment (ganeshkumar1989) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/termsOfUse/translations/Messages_fr_FR.json]: This is addressed by combing them into a single translation

**Comment by ganeshkumar1989** [comment]
> Could you be more explicit in the commit message ?

This is done. Thanks :)

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/termsOfUse/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -2,6 +2,6 @@
   "dedicatedCloud_datacenter_backup_terms_of_use_description1_new": "Êtes-vous sûr(e) de vouloir activer la sauvegarde dans le datacenter {{datacenter}} ?",
   "dedicatedCloud_datacenter_backup_terms_of_use_description2_new": "Attention, vous serez facturé(e) selon le nombre et la taille des machines virtuelles sauvegardées.",
   "dedicatedCloud_datacenter_backup_terms_of_use_tariff_url": "Consulter en détail la nouvelle gamme Veeam Backup",
-  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions": "J’ai lu et j’approuve les conditions générales d’utilisation",
+  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions": "J’ai lu et j’approuve les <a target='_blank' href='{{url}}'>conditions générales d’utilisation</a>",
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/termsOfUse/translations/Messages_fr_FR.json]:
```suggestion
  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions": "J’ai lu et j’approuve les <a target='_blank' href='{{url}} rel="noopener"'>conditions générales d’utilisation</a>",
```

**Pull request merged**